ocs_ci/deployment/rosa.py

```diff
@@ -75,7 +75,11 @@ def destroy(self, log_level="DEBUG"):
             log_level (str): log level openshift-installer (default: DEBUG)
 
         """
+        cluster_details = ocm.get_cluster_details(self.cluster_name)
+        cluster_id = cluster_details.get("id")
         ocm.destroy_cluster(self.cluster_name)
+        rosa.delete_operator_roles(cluster_id)
```

Comment:
I see that this fails in teardown:
```
2022-01-03 15:22:09,749 - MainThread - INFO - ocs_ci.utility.utils.exec_cmd.487 - Executing command: rosa delete operator-roles -c 1pg8nrflqcatuqkq9n39vlspavcnpp2l
2022-01-03 15:22:09,960 - MainThread - DEBUG - ocs_ci.utility.utils.exec_cmd.502 - Command stdout is empty
2022-01-03 15:22:09,961 - MainThread - WARNING - ocs_ci.utility.utils.exec_cmd.506 - Command stderr: [0;31mE:[m Error validating cluster '1pg8nrflqcatuqkq9n39vlspavcnpp2l': There is no cluster with identifier '1pg8nrflqcatuqkq9n39vlspavcnpp2l'

2022-01-03 15:22:09,961 - MainThread - DEBUG - ocs_ci.utility.utils.exec_cmd.509 - Command return code: 1
```
We might need to figure out how to call it correctly.

Reply:
@ebondare I suggest to comment out those 2 new command calls with a note or move it to a different place - I think that we should call those commands during teardown before it ends but we need to test it.